eventmesh-common/src/main/java/org/apache/eventmesh/common/utils/AssertUtils.java

```diff
@@ -27,35 +27,35 @@
 public final class AssertUtils {
 
     /**
-     * assert obj not null
+     * Assert actual is not null
      *
-     * @param obj     obj
+     * @param actual  Object to test
      * @param message error message
      */
-    public static void notNull(final Object obj, final String message) {
-        isTrue(Objects.nonNull(obj), message);
+    public static void notNull(final Object actual, final String message) {
+        isTrue(Objects.nonNull(actual), message);
     }
 
     /**
-     * assert test is true
+     * Assert actual is true
      *
-     * @param test    test
+     * @param actual  boolean to test
      * @param message error message
      */
-    public static void isTrue(final Boolean test, final String message) {
-        if (!Boolean.TRUE.equals(test)) {
+    public static void isTrue(final Boolean actual, final String message) {
+        if (!Boolean.TRUE.equals(actual)) {
             throw new IllegalArgumentException(message);
```

Comment:
If there is no logic in the code that needs to compare the expected value to the actual value, and there is no expected value that needs to be compared, then there is no need to name a single parameter `actual`

Besides, "actual" is an adjective.

The same applies to the other two methods.

Reply:
@Pil0tXia Thanks for your feedback. As the idea of the test is an assertion, naming the variable as `actual `is like naming it `current`. 
Anyway after doing some [research ](https://docs.oracle.com/middleware/11119/jdev/api-reference-esdk/oracle/ide/util/Assert.html#check_boolean_) I saw that for booleans, the variable is usually named `condition`. If this LGTY, I will rename the boolean variable as `condition `and rollback the remaining variables.

Reply:
Yes, `condition` is better. Good idea.